addons/pkg/util/addon_util.go

```diff
@@ -8,16 +8,19 @@ import (
 	"fmt"
 	"net"
 	"strconv"
+	"strings"
 
-	"github.com/pkg/errors"
```

Comment:
this is auto changed by `make fmt` from project root which makes `make go-lint` happy